workbench-core/authorization/src/dynamicAuthorization/models/addUserToGroup.ts

```diff
@@ -3,7 +3,7 @@
  *  SPDX-License-Identifier: Apache-2.0
```

Comment:
Please isolate refactor work such as folder restructuring to a single PR. Do not combine with other work.